lib/jolie-js/src/jolie/js/JsUtils.java

```diff
@@ -121,7 +125,51 @@ public static void valueToJsonString( Value value, boolean extendedRoot, Type ty
 			builder.append( '}' );
 		}
 	}
+	public static void valueToNdJsonString( Value value, boolean extendedRoot, Type type, StringBuilder builder ) throws IOException
+	{
+		Map<String, ValueVector> children = value.children();
+		for( Map.Entry<String, ValueVector> child : children.entrySet() ) {
+			for( Value v : child.getValue() ) {
+				if ( v.hasChildren( JSONARRAY_KEY ) ) {
```

Comment:
Why don't you just call `valueToJsonString` for each item here?

Reply:
because is a public method I didn't think you would like it